packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts

```diff
@@ -0,0 +1,426 @@
+              color: 'HighlightText'
+            }
+          }
+        }
+      }
+    },
+
+    compact && cellCompactStyles
+  ];
+
+  const checkCompactStyles: IStyle = {
+    padding: values.rowHorizontalPadding
+  };
+
+  return {
+    root: [
+      classNames.root,
+      AnimationClassNames.fadeIn400,
+      droppingClassName,
+      anySelected && checkClassNames && checkClassNames.anySelected,
+      isCheckVisible && checkClassNames && checkClassNames.isVisible,
+      isCheckVisible && classNames.isCheckVisible,
+      getFocusStyle(theme, 0, undefined, undefined, isSelected ? neutralSecondary : themePrimary),
+      {
+        borderBottom: `1px solid ${neutralLighter}`,
+        background: colors.defaultBackgroundColor,
+        color: colors.defaultMetaTextColor,
+        display: 'flex',
+        minWidth: '100%',
+        minHeight: values.rowHeight,
+        whiteSpace: 'nowrap',
+        padding: 0,
+        boxSizing: 'border-box',
+        verticalAlign: 'top',
+        textAlign: 'left',
+        selectors: {
+          [`${classNames.listCellFirstChild} &:before`]: {
+            display: 'none'
+          },
+
+          '&:hover': {
+            background: colors.hoverColorBackground,
+            color: colors.hoverMetaTextColor
+          }
+        }
+      },
+      isSelected && selectedStyles,
+      !canSelect && cannotSelectStyles,
+      compact && rootCompactStyles,
+      className
+    ],
+
+    cell: defaultCellStyles,
+    cellMeasurer: [
+      classNames.cellMeasurer,
+      {
+        overflow: 'visible',
+        whiteSpace: 'nowrap'
+      }
+    ],
+    check: [compact && checkCompactStyles],
+    checkCell: [
+      defaultCellStyles,
+      classNames.cellCheck,
+      checkboxCellClassName,
+      {
+        padding: 0,
+        // Ensure that the check cell covers the top border of the cell.
+        // This ensures the click target does not leave a spot which would
+        // cause other items to be deselected.
+        paddingTop: 1,
+        marginTop: -1,
+        flexShrink: 0
+      }
+    ],
+    checkCover: [
+      {
+        position: 'absolute',
+        top: -1,
+        left: 0,
+        bottom: 0,
+        right: 0,
+        display: 'none'
+      },
+
+      anySelected && {
+        display: 'block'
+      }
+    ],
+    fields: [
+      classNames.fields,
+      {
+        display: 'flex',
+        alignItems: 'stretch'
+      }
+    ],
+    isRowHeader: [
+      {
+        color: colors.defaultHeaderTextColor,
+        fontSize: FontSizes.medium
+      }
+    ],
+    isMultiline: [
+      {
+        whiteSpace: 'normal',
+        wordBreak: 'break-word',
+        textOverflow: 'clip'
+      }
+    ],
+    isPadded: [
+      {
+        paddingRight: values.isPaddedMargin + values.rowHorizontalPadding,
+        selectors: {
+          '&.$checkCell': {
+            paddingRight: 0
+          }
+        }
+      }
+    ],
+    shimmer: [],
+    shimmerIconPlaceholder: [],
```

Comment:
Unused or future PR?

Reply:
it is used with the `$shimmer` inside the selectors